program/tests/tests/add_remove_validator.rs

```diff
@@ -43,11 +48,57 @@ async fn test_remove_validator_without_unclaimed_credits() {
     let vote_account = solido.validators.entries[0].pubkey;
     assert_eq!(solido.validators.entries[0].entry.fee_credit, StLamports(0));
 
-    context
-        .try_remove_validator(vote_account)
-        .await
-        .expect("Failed to remove validator.");
+    let result = context.try_remove_validator(vote_account).await;
 
-    let solido = context.get_solido().await;
-    assert_eq!(solido.validators.len(), 0);
+    assert_solido_error!(result, LidoError::ValidatorIsStillActive);
```

Comment:
The test name is misleading now, removing a an _inactive_ validator without unclaimed credits should succeed ... if it fails on being active, then the fact that it has no unclaimed credits isn’t really relevant any more. Also, there is no test now that tests successful removal of the validator. I think we can extend this test to

* After asserting that it has no unclaimed fees, deactivate the validator.
* Then remove it, and assert that that succeeds.

And then we can add a new test to confirm that trying to remove an active validator fails.